samples/matter/light_bulb/sample.yaml

```diff
@@ -91,3 +90,10 @@ tests:
     integration_platforms:
       - nrf7002dk/nrf5340/cpuapp
     tags: sysbuild ci_samples_matter
+  sample.matter.light_bulb.mgrt_dac:
```

Comment:
As for DAC migration. This should be (eventually) enabled by default for nRF54L with and without TF-M. Thus i would consider if we want to have such a name here.

Reply:
We created a separate one record, due to fact that for matter project the DAC migration is an optional feature.
In TAS if we define the non Matter default configuration we would like to keep the pattern per samples.
If we decide the enable DAC migration for all samples and platform, we will assume that the DAC migration feature is an default configuration so it may land within `.debug` configuration as it is for factory data.